### ISSUE TITLE
Java main method run/debug is overridden by Lambda run/debug

### ISSUE THREAD

**Issue by zhangzhx** [aws/aws-toolkit-jetbrains#333]
**Describe the bug**

**To reproduce**

Create a standard Java project with a main method. Click the gutter icon to the left of the main method to run it as shown in the screenshot. 

**Expected behavior**

Expecting to simply run the main method but showing the Lambda run configuration UI.

**Screenshots**

<img width="404" alt="screen shot 2018-10-23 at 11 43 34 am" src="https://user-images.githubusercontent.com/7167513/47383168-ea6ae280-d6b8-11e8-9626-92231b4b61bd.png">

**Your Environment**
 - OS: 
 - JetBrains' Product:
 - JetBrains' Product Version:
 - Toolkit Version:

**Additional context**

**Comment by kiiadi** [comment]
Should we mark a main method?

**Comment by zhangzhx** [comment]
We shouldn't mark main method that's for sure, and we have fixed that. But this is a separate issue. It's we messed up the run configuration for Lambda and Application.

A similar symptom is that when the target class implements Lambda interface and also has main method, the Lambda gutter icon and the standard run gutter icon on the class level mess up. Ie, when you hit Lambda run, it runs the main method or vise versa depending on whomever runs first.
